app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -19,6 +21,9 @@ case class Project(
   description: String,
   visibility: Visibility,
   tileVisibility: Visibility,
+  isAOIProject: Boolean,
+  aoiCadence: Long, /* Milliseconds */
```

Comment:
Well, we either have to add logic which validates the object, or have a less user friendly API. The checking logic wouldn't be very complicated, so I'd personally err on the side of being user friendly over absolutely correct.